src/2dplane/GridStateSpace.cpp

```diff
@@ -10,12 +9,39 @@ using namespace std;
 GridStateSpace::GridStateSpace(float width, float height, int discretizedWidth, int discretizedHeight):
     PlaneStateSpace(width, height),
     _obstacleGrid(width, height, discretizedWidth, discretizedHeight) {
+    setMinStepSize(.1);
+    setMaxStepSize(5);
+    setDistScale(1);
 }
 
 bool GridStateSpace::stateValid(const Vector2f &pt) const {
     return PlaneStateSpace::stateValid(pt) && !_obstacleGrid.obstacleAt(_obstacleGrid.gridSquareForLocation(pt));
 }
 
+Vector2f GridStateSpace::intermediateState(const Vector2f &source, const Vector2f &target, float prevStepSize, float ascGrowthRate, float defaultStepSize) const {
```

Comment:
Is there a need to have a min, max and default step size?  I wonder if we could achieve the same thing with just two of those.  Maybe have the regular `stepSize` be the min (since I don't think we don't really want smaller steps than a regular RRT), then have a separate `maxStepSize` property.  This way we can remove the `minStepSize` and still get the same end result.  Thoughts?